charts/eg/templates/certgen.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "eg.fullname" . }}-certgen
+  labels:
+  {{- include "eg.labels" . | nindent 4 }}
+spec:
+  backoffLimit: 1
+  completions: 1
+  parallelism: 1
+  template:
+    metadata:
+      labels:
+        app: {{ include "eg.fullname" . }}-certgen
+    spec:
+      containers:
+      - command:
+        - envoy-gateway
+        - certgen
+        image: docker.io/envoyproxy/gateway-dev:latest
```

Comment:
Should this be templated?

Reply:
I'd suggest templating it from the chart app version field.